docs/sources/get-started/labels/_index.md

```diff
@@ -9,83 +9,283 @@ aliases:
 
-If you are familiar with Prometheus, the term used there is series; however, Prometheus has an additional dimension: metric name. Loki simplifies this in that there are no metric names, just labels, and we decided to use streams instead of series.
+- Explore Logs
+- Grafana Cloud Application Observability
 
 {{< admonition type="note" >}}
-Structured metadata do not define a stream, but are metadata attached to a log line.
-See [structured metadata]({{< relref "./structured-metadata" >}}) for more information.
+If you are already applying a `service_name`, Loki will use that value.
 {{< /admonition >}}
 
-## Format
+Loki will attempt to create the `service_name` label by looking for the following labels in this order:
+
+- service_name
+- service
+- app
+- application
+- name
+- app_kubernetes_io_name
+- container
+- container_name
+- component
+- workload
+- job
+
+If no label is found matching the list, a value of `unknown_service` is applied.
+
+You can change this list by providing a list of labels to `discover_service_name` in the [limits_config](/docs/loki/<LOKI_VERSION>/configure/#limits_config) block.  If you are using Grafana Cloud, contact support to configure this setting.
+
+## Default labels for OpenTelemetry
+
+If you are using either Grafana Alloy or the OpenTelemetry Collector as your Loki client, then Loki automatically assigns some of the OTel resource attributes as labels. Resource attributes map well to index labels in Loki, since both usually identify the source of the logs.
+
+By default, the following resource attributes will be stored as labels, with periods (`.`) replaced with underscores (`_`), while the remaining attributes are stored as [structured metadata](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/structured-metadata/) with each log entry:
+
+- cloud.availability_zone
+- cloud.region
+- container.name
+- deployment.environment
+- k8s.cluster.name
+- k8s.container.name
+- k8s.cronjob.name
+- k8s.daemonset.name
+- k8s.deployment.name
+- k8s.job.name
+- k8s.namespace.name
+- k8s.pod.name
+- k8s.replicaset.name
+- k8s.statefulset.name
+- service.instance.id
+- service.name
+- service.namespace
+
+{{% admonition type="note" %}}
+Because Loki has a default limit of 15 index labels, we recommend storing only select resource attributes as labels. Although the default config selects more than 15 Resource Attributes, some are mutually exclusive.
+{{% /admonition %}}
+
+{{< admonition type="tip" >}}
+For Grafana Cloud Logs, see the [current OpenTelemetry guidance](https://grafana.com/docs/grafana-cloud/send-data/otlp/otlp-format-considerations/#logs).
+{{< /admonition >}}
+
+The default list of Resource Attributes to store as labels can be configured using `default_resource_attributes_as_index_labels` under the [distributor's otlp_config](https://grafana.com/docs/loki/<LOKI_VERSION>/configure/#distributor).  If you are using Grafana Cloud, contact support to configure this setting.
```

Comment:
I am not sure if this has changed but you can also promote and exclude labels otel via the `limits_config`
```
limits_config:
  otlp_config:
```